esx_service/vmdk_ops.py

```diff
@@ -847,16 +848,31 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
     if not datastore:
         datastore_url = default_datastore_url
         datastore = default_datastore
+        use_default_ds = True
```

Comment:
Because the user thinks they can create volumes on all the "known datastores" and when they try they get an error saying they have no privilege. Best show only those datastores that are actually "accessible to the tenant".